sponge_docs_theme/static/spongie-mark.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can directly link to the Spongie SVG on our homepage, no need to copy it to `sponge_docs_theme`. (We do the same for Ore and the Forums).